### ISSUE TITLE
=dev-libs/icu-70*:0 required by www-client/ungoogled-chromium-bin

### ISSUE THREAD

**Issue by zendet** [PF4Public/gentoo-overlay#161]
WARNING: One or more updates/rebuilds have been skipped due to a dependency conflict:
dev-libs/icu:0
  (dev-libs/icu-71.1-r1:0/71.1::gentoo, ebuild scheduled for merge) USE="-debug -doc -examples -static-libs -test -verify-sig" ABI_X86="32 (64) (-x32)" conflicts with  =dev-libs/icu-70*:0 required by (www-client/ungoogled-chromium-bin-102.0.5005.115:0/0::pf4public, installed) USE="generic -convert-dict -core2 -haswell" ABI_X86="(64)" L10N="am ar bg bn ca cs da de el en-GB es es-419 et fa fi fil fr gu he hi hr hu id it ja kn ko lt lv ml mr ms nb nl pl pt-BR pt-PT ro ru sk sl sr sv sw ta te th tr uk vi zh-CN zh-TW"

in short: became impossible upgrade icu to 71.1-r1 because is hardcoded to ebuild

**Comment by zendet** [comment]
cool
